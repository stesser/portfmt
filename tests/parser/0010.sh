echo ".if 1" | ${PORTFMT} -d 2>&1 | diff -L "0010.expected" -L "0010.actual" -u 0010.expected -
