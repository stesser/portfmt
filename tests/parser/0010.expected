portfmt: error building AST: node stack not exhausted
